tutorials/physics/collision_shapes_3d.rst

```diff
@@ -0,0 +1,133 @@
+
+- :ref:`class_BoxShape`
+- :ref:`class_SphereShape`
+- :ref:`class_CapsuleShape`
+- :ref:`class_CylinderShape` (only when using the Bullet physics engine)
+
+Using one or several primitive collision shapes, you can represent the collision
+of most smaller objects well. However, for more complex objects such as a large
+ship or a whole level, you may want to use convex shapes or concave shapes
+instead (see below).
+
+Primitive shapes are recommended for dynamic objects such as RigidBodies and
+KinematicBodies, as their behavior usually is the most reliable and predictable.
+On top of that, they often provide better performance as well.
+
+Convex collision shapes
+-----------------------
+
+:ref:`Convex collision shapes <class_ConvexPolygonShape>` are a compromise
+between primitive collision shapes and concave collision shapes. They can
+represent shapes of any complexity, but with an imporatant caveat. As their name
+implies, an individual shape can only represent a *convex* shape. For instance,
+a pyramid is *convex* but an hollow box is *concave*. To represent a concave
+object with a single collision shape, you need to use a concave collision shape
+(see below).
+
+However, depending on the object's complexity, you may get better performance by
+using multiple convex shapes instead of a concave collision shape. This is why
+Godot lets you make use of *convex decomposition* to generate convex shapes that
+roughly match a concave object. That said, this performance advantage no longer
+applies after a certain amount of convex shapes. Due to this, for large and
+complex objects such as a whole level, it's recommend to use concave shapes
+instead.
+
+You can generate one or several convex collision shapes from the editor by
+selecting a MeshInstance and using the **Mesh** menu at the top of the 3D
+viewport. The editor exposes two generation modes:
+
+- **Create Single Convex Collision Sibling** uses the Quickhull algorithm.
+  It creates one CollisionShape node with an automatically generated convex
+  collision shape. Since it only generates a single shape, it provides better
+  performance and is recommended for small objects.
+
+- **Create Multiple Convex Collision Siblings** uses the V-HACD algorithm.
+  It creates several CollisionShape nodes, each with their own convex collision
+  shape. Since it genereates multiple shapes, it is more accurate for concave
+  objects at the cost of performance. For objects with medium complexity, it
+  will likely be faster than using a single concave collision shape.
+
+Concave (trimesh) collision shapes
+----------------------------------
+
+:ref:`Concave collision shapes <class_ConcavePolygonShape>` (also called trimesh
+collision shapes) can take any form, from a few triangles to thousands of
+triangles. Concave shapes are the slowest option, but are also the most accurate
+collision shapes available in Godot. **Concave shapes can only be used within
```

Comment:
Yes they are accurate in the sense that they follows perfectly the mesh. Though, the contacts results are not so precise like would be for a Primitive shape.

Not sure how to improve this sentence, though.